src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(c_variadic, ptr_wrapping_offset_from, ptr_cast)]
+#![feature(c_variadic, ptr_wrapping_offset_from, ptr_cast, drain_filter)]
```

Comment:
I know `drain_filter` is a nice feature, but I really want to get onto stable eventually, so lets not introduce new nightly dependencies where we have to.

Reply:
Fixed